pkg/crossplane/service_redis.go

```diff
@@ -79,6 +79,15 @@ func (rsb RedisServiceBinder) GetBinding(ctx context.Context, bindingID string)
 			},
 		},
 	}
+	mbytes, ok := s.Data[MetricsPortKey]
+	if ok {
+		mp := string(mbytes)
+		creds["metrics"] = []string{
```

Comment:
Are you purposely not checking for an empty port (`mp`) here, as you did in the other file?